ssat/notebooks/bayesian_example.ipynb

```diff
@@ -136,7 +137,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "match_df = pd.read_parquet(\"ssat/data/sample_handball_match_data.parquet\")\n",
```

Comment:
This will not work for users installing the `ssat` package. Only for developers installing the repository.